src/urllib3/connection.py

```diff
@@ -255,10 +255,11 @@ def putheader(self, header: str, *values: str) -> None:
         if not any(isinstance(v, str) and v == SKIP_HEADER for v in values):
             super().putheader(header, *values)
         elif to_str(header.lower()) not in SKIPPABLE_HEADERS:
+            skippable_headers = "', '".join(
```

Comment:
I would also be fine with having it inline in the f-string. Also would be fine with sticking to `map`, apparently if it does not use a `lambda` it is faster than list comprehension, which IMO is more readable and more pythonic than `map`.

Reply:
This is more readable, yes 👍